src/pages/profiles/ProfileList.tsx

```diff
@@ -114,7 +114,10 @@ const ProfileList: FC = () => {
       columns: [
         {
           content: (
-            <div className="u-truncate" title={profile.name}>
+            <div
+              className="u-truncate"
+              title={`link to profile ${profile.name}`}
```

Comment:
Here we have a list of profiles, so just the "profile.name" is probably the right thing to use?